### DISCUSSION TITLE
Can micronaut-serialization be used independently?

### DISCUSSION THREAD

**Discussion by sixwaaaay** [micronaut-projects/micronaut-data#2667] [Q&A]
can I run this reflection-free serialization library without micronaut core?

**Comment by sdelamo** [comment]
Almost every feature depends on parts of Micronaut Core. Not sure exactly what you want to achieve.

**Reply by sixwaaaay** [reply]
a json serializer without reflection

**Comment by graemerocher** [comment]
Yes you can use it independently. The ObjectMapper interface has some static factory methods to create an instance.

The Micronaut annotation processors need to be on the annotation processor classpath
